src/ast/ddl.rs

```diff
@@ -39,6 +39,27 @@ use crate::ast::{
 use crate::keywords::Keyword;
 use crate::tokenizer::Token;
 
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
```

Comment:
It could be helpful here to add a description comment for the enum with a link to the docs